google-home-community.groovy

```diff
@@ -2520,10 +2533,8 @@ private deviceStateForTrait_Brightness(deviceTrait, device) {
 @SuppressWarnings(['UnusedPrivateMethod', 'UnusedPrivateMethodParameter'])
 private deviceStateForTrait_CameraStream(deviceTrait, device) {
     return [
-        cameraStreamAccessUrl:        device.currentValue(deviceTrait.cameraStreamAttribute),
-        cameraStreamReceiverAppId:    null,
-        cameraStreamAuthToken:        null,
-        cameraStreamProtocol:         deviceTrait.cameraStreamSupportedProtocols
+        cameraStreamAccessUrl:        device.currentValue(deviceTrait.cameraStreamURLAttribute),
+        cameraStreamProtocol:         device.currentValue(deviceTrait.cameraStreamProtocolAttribute),
```

Comment:
Are these needed?  The documentation doesn't list any states for the CameraStream trait.

Reply:
You are correct, not needed so they are now removed.